README.md

```diff
@@ -124,11 +133,16 @@ Change the paths to your git repos
 }
 ```
 
-Once you have `~/.rizzo.json`, change to your top level control repository and generate your `Vagrantfile`.
+Once you have a personal config file, `~/.rizzo.json`, change directories to
+your top level control repository and generate your `Vagrantfile`:
 
 ```shell
+cd ~/git/puppetdata
```

Comment:
`puppet-control-myteam` instead of puppetdata as we want to encourage team's to have their own and layer on top of each other.

Reply:
Fixed, added a third entry in the example above to match up.